### PR TITLE
(IMAGES-485) Ensure WinRM is started post-clone

### PR DESCRIPTION
**Pull request by jcoconnor**
WinRM gets disabled by Packer as part of its shutdown/restart
processing. Ensure it is enabled during the post-clone configuration on
vmpooler and vagrant.

Also include 
1. Add ```vagrantcloud.yaml``` file for vagrant images (IMAGES-366)
2. Disable error checking during Temp file delete (IMAGES-484)
3. Add ```template_config``` to the output directory and image name (pipeline issues esp for vagrant)

### CODE REVIEW

File: scripts/windows/clean-disk-dism.ps1 [hunk 1]
```diff
@@ -60,6 +60,8 @@ net stop wuauserv
 Set-Service wuauserv -StartupType Disabled
 
 # Clean up files (including those not addressed by cleanmgr)
+# Disable error checking as this sometimes throws errors that can be safely ignored (mostly locked files)
```
Comment (glennsarti) [inline scripts/windows/clean-disk-dism.ps1]: You can use try { } catch { } to ignore errors here as well.  Not 100% if that's PS 2 compatible

Comment (jcoconnor) [inline scripts/windows/clean-disk-dism.ps1]: Done

**Comment by jcoconnor** [comment]
Putting this to don't merge as need to fix up some directory naming issues that are breaking the pipeline.

**Comment by jcoconnor** [comment]
@puppetlabs/agent-platform-team This PR is ready for merge.

Directory naming issues mentioned earlier are resolved and builds run clean.

**Comment by McdonaldSeanp** [comment]
👍

**Pull request merged**